bodhi/server/config.py

```diff
@@ -407,6 +407,12 @@ class BodhiConfig(dict):
         'initial_bug_msg': {
             'value': '%s has been submitted as an update to %s. %s',
             'validator': unicode},
+        'greenwave_api_url': {
+            'value': 'https://greenwave.fedoraproject.org/api/v1.0',
+            'validator': unicode},
+        'greenwave_enabled': {
+            'value': False,
+            'validator': _validate_bool},
```

Comment:
The reason I keep the code that blocks updates based on test results is that I assume someone must have used it and I do not want to break them. Eventually, we will get rid of it for sure. 
Yeah, I agree that we should reuse the ci.enabled setting, but I would suggest using a different name to make it more clearly.  What about 'gating.enabled' or 'test_gating.enabled'?